src/Akka.Persistence.Sql/Db/AkkaPersistenceDataConnectionFactory.cs

```diff
@@ -136,13 +136,24 @@ public AkkaPersistenceDataConnectionFactory(IProviderConfig<SnapshotTableConfigu
                 .Member(r => r.Timestamp)
                 .HasColumnName(columnNames.Created)
 
-                // TODO: Disabling this for now, will need a migration script to support this
-                .Member(r => r.WriteUuid)
-                .IsNotColumn()
-
                 .Member(r => r.TagArr)
                 .IsNotColumn();
 
+            if (journalConfig.TableMapName == "default")
```

Comment:
We only create the `WriterUuid` column if we're creating a new default database (`Akka.Persistence.Sql` only), and we do not offer any backward compatibility check.

Question: Should this feature be gated using HOCON configuration instead? let the user decide if they want to use WriterUuid and use `true` as default?